src/ORM/Table.php

```diff
@@ -1213,15 +1213,15 @@ public function get($primaryKey, $options = [])
      * called allowing you to define additional default values. The new
      * entity will be saved and returned.
      *
-     * @param array $search The criteria to find existing records by.
+     * @param array|\Cake\ORM\Query $search The criteria to find existing records by.
```

Comment:
It would be good to note that when using a query to find, no properties will be set in the new record.